service-general/src/main/java/sjchat/general/GRPCServer.java

```diff
@@ -24,7 +24,6 @@ public void run() {
         System.out.println("*** server shut down");
       }
     });
-    blockUntilShutdown();
```

Comment:
It was necessary in an earlier state of the web socket implementation but since it is no longer needed I'll put it back.